README.md

```diff
@@ -163,6 +163,7 @@ the one below:
   
     curl -v -X PUT "http://localhost:9650/field/application/<YOUR_CLIENT_ID_HERE>/context/<YOUR_APP_CONTEXT_HERE>/version/<YOUR_VERSION_HERE>" -H "Cache-Control: no-cache" -H "Content-Type: application/json" -d '{ "fields" : { "callbackUrl" : "http://localhost:9826/customs-notifications-receiver-stub/pushnotifications", "securityToken" : "aaaa01f9-ec3b-4ede-b263-61b626dde232" } }'
 
+
```

Comment:
it seems we don't need Caucasian space PRs any more:

You can trigger a change with `git commit --allow-empty` instead of making a whitespace change

source: https://hmrcdigital.slack.com/archives/G830N2RML/p1534149409000145